tests/unit/apps/test_subordinate_auxiliary.py

```diff
@@ -0,0 +1,78 @@
+#  Copyright 2023 Canonical Limited
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Tests of the Auxiliary Subordinate application class."""
+
+from cou.apps.subordinate_auxiliary import OpenStackAuxiliarySubordinateApplication
+from cou.steps import UpgradeStep
+from cou.utils.openstack import OpenStackRelease
+from tests.unit.apps.utils import assert_plan_description
+
+
+def test_subordinate_auxiliary(status):
```

Comment:
Since we are using `auxiliary subordinate` everywhere else, I think we should keep it consistent in the tests.
```suggestion
def test_auxiliary_subordinate(status):
```